website/src/controllers/groups.js

```diff
@@ -136,8 +136,29 @@ api.get = function(req, res, next) {
   populateQuery(gid, q);
   q.exec(function(err, group){
     if (err) return next(err);
-    if (!group && gid!=='party') return res.json(404,{err: "Group not found or you don't have access."});
-    res.json(group);
+    if (!group && group != null && gid!=='party') return res.json(404,{err: "Group not found or you don't have access."});
```

Comment:
@TheHollidayInn could you remove the `group != null` check? it's redundant with !group